django/utils/six.py

```diff
@@ -852,3 +854,34 @@ def python_2_unicode_compatible(klass):
     else:
         memoryview = buffer
     buffer_types = (bytearray, memoryview)
+
+
+# backport of Python 3.4's glob.escape
```

Comment:
I'd rather put this in a separate module like `django.utils.glob` unless you plan to try to get it included in six itself.